pandas/_libs/tslibs/timestamps.pyx

```diff
@@ -308,9 +310,16 @@ cdef class _Timestamp(ABCTimestamp):
 
         elif not isinstance(self, _Timestamp):
             # cython semantics, args have been switched and this is __radd__
+            # TODO: Cython 3, remove this it moved to __radd__
```

Comment:
nitpick for grepping purposes we've tried to standardize on "# TODO(cython3): foo bar baz"